cmd/aws-health-notifier/main.go

```diff
@@ -41,6 +41,10 @@ func sendNotification(event events.CloudWatchEvent) {
 		logger.Fatal("failed to decrypt slackWebhookURL", zap.Error(err))
 	}
 	slack := slackhook.New(slackWebhookURL)
+	description := "no description found in health check"
+	if len(health.Description) > 0 {
```

Comment:
I don't understand the difference between `health.Description[0].Latest` and `health.Description`.

Reply:
Looks like the event has this structure: https://github.com/trussworks/truss-aws-tools/blob/master/pkg/awshealth/aws_health_event.go#L18.

The `Description` is a slice of these https://github.com/trussworks/truss-aws-tools/blob/master/pkg/awshealth/aws_health_event.go#L11-L14.  

So I hope what I'm checking is that if there are no event descriptions then I'll not be trying to push a description into the new struct and avoid the "index out of range" problem.